components/docs-chef-io/content/automate/saml.md

```diff
@@ -89,7 +89,17 @@ where:
 - `sso_url` contains the value of _Login URL_
 - `entity_issuer` contains the value of _Identifier (Entity ID)_
 
-See the SAML Configuration Settings below for further configuration options.
+See the [SAML Configuration Settings]({{< relref "saml.md#saml-configuration-settings" >}}) for further configuration options.
+
+{{< warning >}}
+Azure AD lets you chose the _NameID_ field, and optionally apply _transformations_ to it.
+The SAML configuration setting is only respected if the selected value in "Choose name identifier format" in Azure AD matches the `name_id_policy_format` configuration in Chef Automate.
+
+For example, when selecting the specific user attribute `user.email` for NameID in Azure AD:
+
+1. In Azure AD for "Choose name identifier format", select "EmailAddress", and configure the attribute and (optional) transformation.
```

Comment:
@srenatus Are there more details about configuring the attribute and transformation to add for this example?

Reply:
With discussion, this question has been answered! We're keeping this warning simple.